nix/haskell.nix

```diff
@@ -104,6 +104,17 @@ let
               export TMPDIR=/tmp
             '';
 
+          # Attempt to ensure visible progress in the macOS hydra job.
+          #
+          # An hypothesis is that #2472 is caused by heavy load and unfocused
+          # resources from running the tests concurrently, risking that the slowest
+          # hspec runner - and thererefore the stdout - being silent for 900s causing
+          # hydra to timeout.
+          #
+          # Setting -j 1 should hopefully focus the resource we have in one place. It
+          # should go silent less often, at the expense of the full run getting slower.
+          unit.testFlags = if pkgs.stdenv.hostPlatform.isDarwin then ["-j" "1"] else [];
```

Comment:
This is setting the `unit` test-suite flags for the `cardano-wallet` package, but I think you want the `cardano-wallet-core` package, right?